app/models/ability.rb

```diff
@@ -14,14 +14,18 @@ def initialize(user)
       can :read, User
       can :read, Test
       can :read, TestSetting
+      can [:edit, :update, :destroy], MethodicalWork, :user_id => user.id
+      can [:read, :create], MethodicalWork
```

Comment:
why do we need this splitting? Could we say `can :manage, MethodicalWork` or with `user_id` ? Please try and check